libmusic/src/chord_tpl.cpp

```diff
@@ -249,4 +250,14 @@ size_t ChordTpl::SlashSubtypesCnt(chord_quality_t q)
     return chord_qlty_tpls_[q].size();
 }
 
+ostream& operator<<(std::ostream& os, const ChordTpl& tpl)
+{
+    os << Chord(tpl.root_note_, tpl.chord_quality_) << " : ";
+    for (uint8_t i = 0; i < tpl.tpl_.size(); i++) {
```

Comment:
```suggestion
    for (auto i = 0; i < tpl.tpl_.size(); i++) {
```

Reply:
auto goes for a signed int which fails compilation due to comparison with unsigned `tpl.tpl_.size()`